.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "singleQuote": true,
+  "trailingComma": "none",
+  "bracketSpacing": false,
+  "jsxBracketSameLine": true
+}
```

Comment:
@Spetastian Let's require a newline at the end of every file -- .js*, .*rc, etc; everything but binaries